apps/ledger-live-desktop/tests/fixtures/common.ts

```diff
@@ -175,6 +175,9 @@ export const test = base.extend<TestFixtures>({
     await page.waitForLoadState("domcontentloaded");
     await page.waitForSelector("#loader-container", { state: "hidden" });
 
+    // Click on accept analytics button if it exists
+    page.getByTestId("accept-analytics-button").click();
```

Comment:
but if this button doesn't exist then the test will fail, is it expected?

Reply:
Thank you for the review. No it's not expected. I wrongly assumed the flag was going to be removed for some reason. I just added a try catch. It's not ideal as it would slow down the tests when the flag is off but I can't think of another solution.

Reply:
would something that work for you ?👇 
```
private acceptAnalyticsButton = page.getByTestId("accept-analytics-button");

const analyticsVisible = await acceptAnalyticsButton.isVisible();
if (analyticsVisible) {
        await acceptAnalyticsButton.click();
} 
```